app/models/conversion_host/configurations.rb

```diff
@@ -26,7 +26,21 @@ def queue_configuration(op, instance_id, resource, params, auth_user = nil)
         :args        => [params, auth_user]
       }
 
-      MiqTask.generic_action_with_callback(task_opts, queue_opts)
+      task_id = MiqTask.generic_action_with_callback(task_opts, queue_opts)
+
+      # Set the context_data after the fact because the above call only accepts
+      # certain options while ignoring the rest. We also don't want to store
+      # any ssh key information. Useful for a retry option in the UI, and
+      # informational purposes in general.
+      #
+      MiqTask.find(task_id).tap do |task|
```

Comment:
@djberg96 so you want to use the old method so this can be backported then followup with a refactoring PR to just pass in the context_data?

Reply:
That's the plan, yes.